pkg/api/api_test.go

```diff
@@ -0,0 +1,30 @@
+package api
+
+import (
+	"net/http"
+	"net/http/httptest"
+	"strings"
+	"testing"
+)
+
+func TestApi(t *testing.T) {
+	ts := httptest.NewServer(NewRouter())
+	defer ts.Close()
+
+	_, err := http.Get(strings.Join([]string{ts.URL, "api/status"}, "/"))
+	if err != nil {
+		t.Fatal(err)
+	}
+
+}
+
+func TestQuery(t *testing.T) {
+	ts := httptest.NewServer(NewRouter())
+	defer ts.Close()
+
+	_, err := http.Get(strings.Join([]string{ts.URL, "api/query"}, "/"))
```

Comment:
Just so you know, you can isolate err to a scope and do this in one block like this:
```go
if _, err := http.Get(strings.Join([]string{ts.URL, "api/query"}, "/")); err != nil {
    t.Fatal(err)
}
```